### PR TITLE
Add OpenAI.rough_token_count class method

### PR DESCRIPTION
**Pull request by jamiemccarthy**
It can be useful to estimate the token count of some text before sending it to OpenAI. This PR adds an `OpenAI.rough_token_count` class method which does this.

To perform the estimation, it uses the two "helpful rules of thumb" in OpenAI's [docs](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them) and takes the arithmetic average.

The empty string returns 0, and any other string returns an integer greater than 0.

This is a rewrite of #306 to remove the Tiktoken functionality and change the interface as requested.

## All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](../blob/main/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: lib/openai.rb [hunk 1]
```diff
@@ -52,4 +52,14 @@ def self.configuration
   def self.configure
     yield(configuration)
   end
+
+  def self.rough_token_count(content = "")
```
Comment (alexrudall) [inline lib/openai.rb:56]: Can you add your explanation from the PR description here

```suggestion

  # Explain what this does
  # Link to OpenAI docs
  def self.rough_token_count(content = "")
```

**Review by alexrudall — CHANGES_REQUESTED** [review]
This is great, thanks a lot for rewriting. Just some documentation needed, would you mind adding a section to the README with a couple lines briefly explaining how to estimate tokens in a string and link to the OpenAI docs. Probably between Azure and Models sections I think? 

And couple inline comments also, just with the method.

File: lib/openai.rb (hunk 1 above)
Comment (jamiemccarthy) [reply lib/openai.rb]: Done, in 3845a45a8664b6b0c5f0096d303622ca017e58bd.

If [that new section in the README](https://github.com/alexrudall/ruby-openai/blob/3845a45a8664b6b0c5f0096d303622ca017e58bd/README.md#counting-tokens) is too wordy, let me know and I'll cut it down.

**Comment by alexrudall** [comment]
This is perfect thanks @jamiemccarthy!

**Review by alexrudall — APPROVED** [review]
